### PR TITLE
Add dev script

### PR DESCRIPTION
**Pull request by smcclem**
**What this PR does / why we need it?**:

Add dev.sh script, which adds some quick developer tools. It can build and push the operator to an OCP cluster, and install it with OLM (using the private registry). It can also setup and run the scorecard tests. 

**Does this PR introduce a user-facing change?**

- [ ] User guide
- [ ] `CHANGELOG.md`

**Which issue(s) this PR fixes**:
Fixes #

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -224,19 +253,19 @@ docker-login: ## Log in to a Docker registry.
...
-	docker push ${IMG}
+	$(CONTAINER_COMMAND) push  $(PODMAN_SKIP_TLS_VERIFY) ${IMG}
 
 .PHONY: bundle-build
 bundle-build: ## Build the bundle image.
-	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+	$(CONTAINER_COMMAND) build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
 .PHONY: bundle-push
-bundle-push: docker-login ## Push the bundle image.
```
Comment (idlewis) [inline Makefile:238]: As the following command still might run docker, is it safe to remove the 'docker-login' dependency here?

File: scripts/dev.sh [hunk 2]
```diff
@@ -0,0 +1,539 @@
+#!/bin/bash
```
Comment (idlewis) [inline scripts/dev.sh:1]: This doesn't seem to be checked in with execute permission. Might be helpful to do so.

File: Makefile (hunk 1 above)
Comment (smcclem) [reply Makefile:238]: Both WLO and OLO do not have this dependency here. I am not sure why its here and if there is an actual need for it.

**Review by idlewis — APPROVED** [review]
LGTM

**Pull request merged**